internal/errors/errors.go

```diff
@@ -0,0 +1,63 @@
+package errors
+
+import (
+	errorsmod "cosmossdk.io/errors"
+
+	"github.com/cosmos/ibc-go/v7/modules/core/exported"
+)
+
+const codespace = exported.ModuleName
+
+var (
+	// ErrInvalidSequence is used the sequence number (nonce) is incorrect
+	// for the signature.
+	ErrInvalidSequence = errorsmod.Register(codespace, 1, "invalid sequence")
```

Comment:
A couple years ago, it'd be the case that if you used error code 1, the error message would get redacted and replaced by "internal error". I'm unsure if this has changed, but it would be good to double check

Reply:
Thanks for the heads up I will look into it

Reply:
Looks like it just got changed https://github.com/cosmos/cosmos-sdk/pull/11960/files#diff-cbd3ee4d9e83821aab653ac4a36825cf310ecc8fc8d225024e3f74a305621ba3L124

Reply:
looks like it got fixed in beta7 release https://github.com/cosmos/cosmos-sdk/blob/ce25dd2cc101737959e38fd888bbd4765c14836e/errors/abci.go#L46

Reply:
So I think this should be just fine :+1: